src/pages/Review/TasksReview/TasksReviewTable.js

```diff
@@ -805,6 +856,84 @@ const setupColumnTypes = (props, openComments, data, criteria, pageSize) => {
     },
   }
 
+  columns.metaReviewStatus = {
+    id: 'metaReviewStatus',
+    Header: makeInvertable(props.intl.formatMessage(messages.metaReviewStatusLabel),
+                           () => props.invertField('metaReviewStatus'),
+                           _get(criteria, 'invertFields.metaReviewStatus')),
+    accessor: 'metaReviewStatus',
+    sortable: true,
+    filterable: true,
+    exportable: t => props.intl.formatMessage(messagesByMetaReviewStatus[t.metaReviewStatus]),
+    maxWidth: 180,
+    Cell: props => (_isUndefined(props.value) ? "" :
+      <StatusLabel
+        {...props}
+        intlMessage={messagesByMetaReviewStatus[props.value]}
+        className={`mr-review-${_kebabCase(keysByReviewStatus[props.value])}`}
+      />
+    ),
+    Filter: ({ filter, onChange }) => {
+      const options = [
+      ]
+
+      if (props.reviewTasksType === ReviewTasksType.metaReviewTasks) {
+        options.push(<option key="all" value="0,-2">{props.intl.formatMessage(messages.allNeeded)}</option>)
+        options.push(<option key="none" value="-2">{props.intl.formatMessage(messages.metaUnreviewed)}</option>)
+        options.push(
+          <option key={keysByReviewStatus[TaskReviewStatus.needed]} value={TaskReviewStatus.needed}>
+            {props.intl.formatMessage(messagesByMetaReviewStatus[TaskReviewStatus.needed])}
+          </option>
+        )
+      }
+      else {
+        options.push(<option key="all" value="all">All</option>)
+        options.push(<option key="none" value="-2">{props.intl.formatMessage(messages.metaUnreviewed)}</option>)
+        _each(TaskReviewStatus, (status) => {
+          if (status !== TaskReviewStatus.unnecessary &&
+              isMetaReviewStatus(status)) {
+
+            options.push(
+              <option key={keysByReviewStatus[status]} value={status}>
+                {props.intl.formatMessage(messagesByMetaReviewStatus[status])}
+              </option>
+            )
+          }
+        })
+      }
+
+      return (
+        <select
+          onChange={event => onChange(event.target.value)}
+          style={{ width: '100%' }}
```

Comment:
Does `className="mr-w-full"` not work here? That would be preferred over the `style`